VERSION

```diff
@@ -0,0 +1 @@
+0.3.dev
```

Comment:
dev?

Reply:
this gets replaced in CI to either non (0.3) or RC (0.3-rc) depending on the branch.